eng/pipelines/jobs/windows-build-PR.yml

```diff
@@ -11,11 +11,7 @@ jobs:
     timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
     pool:
       # Use a hosted pool when possible.
-      ${{ if eq(variables['System.TeamProject'], 'public') }}:
```

Comment:
Since we have internal WD PRs, the PR template will probably get used on internal projects.